jinseok/service/userService.js

```diff
@@ -1,8 +1,11 @@
+const { getUserPassword } = require('../model/userDao');
 const userDao = require('../model/userDao');
 const {
   pwValidationCheck,
   emailValidationCheck,
 } = require('../utils/validation-check.js');
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
```

Comment:
```suggestion
const bcrypt = require('bcrypt');
const jwt = require('jsonwebtoken');

const { getUserPassword } = require('../model/userDao');
const userDao = require('../model/userDao');
const {
  pwValidationCheck,
  emailValidationCheck,
} = require('../utils/validation-check.js');
```
모듈을 import 하는 순서에 대한 컨벤션은 built-in 모듈 - 외부 모듈 - custom 모듈 순서입니다!